prov/efa/test/efa_unit_test_ep.c

```diff
@@ -309,7 +310,71 @@ void test_efa_rdm_ep_pkt_pool_page_alignment(struct efa_resource **state)
 
 	fi_close(&ep->fid);
 }
+/**
+ * @brief When using LL128 protocol, test the packet allocated from read_copy_pkt_pool
+ *  is 128 byte aligned.
+ * 
+ * @param[in]	state		struct efa_resource that is managed by the framework
+ */
+void test_efa_rdm_read_copy_pkt_pool_128_alignment(struct efa_resource **state)
+{
+	int ret;
+	struct efa_rdm_pke *pkt_entry;
+	struct fid_ep *ep;
+	struct efa_rdm_ep *efa_rdm_ep;
+	struct efa_resource *resource = *state;
 
+	efa_unit_test_resource_construct(resource, FI_EP_RDM);
+
+	ret = fi_endpoint(resource->domain, resource->info, &ep, NULL);
+	assert_int_equal(ret, 0);
+	efa_rdm_ep = container_of(ep, struct efa_rdm_ep, base_ep.util_ep.ep_fid);
+	efa_rdm_ep->sendrecv_in_order_aligned_128_bytes = 1;
+
+	pkt_entry = efa_rdm_pke_alloc(efa_rdm_ep, efa_rdm_ep->rx_readcopy_pkt_pool,
+					  EFA_RDM_PKE_FROM_READ_COPY_POOL);
+	assert_non_null(pkt_entry);
+	assert(ofi_is_addr_aligned((void *)pkt_entry->wiredata, EFA_RDM_IN_ORDER_ALIGNMENT));
+	efa_rdm_pke_release_rx(pkt_entry);
+
+	fi_close(&ep->fid);
+}
+
+/**
+ * @brief When using LL128 protocol, the copy method is local read. 
+ *
+ * @param[in]	state		struct efa_resource that is managed by the framework
+ */
+void test_efa_rdm_pke_get_available_copy_methods(struct efa_resource **state)
```

Comment:
This only test the available copy methods when LL128 is enabled. You need to change the function name to indicate that